src/components/notifications/notification-settings.tsx

```
@@ -0,0 +1,209 @@
+    notification_enabled: true,
+    email_notifications: true,
+    push_notifications: false,
+    notification_time: '09:00',
+  });
+  const [loading, setLoading] = useState(false);
+  const [saving, setSaving] = useState(false);
+  const supabase = createClient();
+
+  useEffect(() => {
+    fetchSettings();
+  }, [fetchSettings]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)